check50-wrapper

```diff
@@ -15,7 +15,7 @@ tag_sha=$(git commit-tree HEAD^{tree} -m "$tag_name")
 git push origin $tag_sha:refs/tags/$tag_name
 
 # get style50 results
-style50_results=$(unset CHECK50_TOKEN CHECK50_ID; style50 --ignore "*.git*" --output=score .)
+style50_results=$(unset CHECK50_TOKEN CHECK50_ID; style50 --ignore "*.git*" --ignore "*~c9*" --output=score .)
```

Comment:
tbh shouldn't `style50` just ignore all dotfiles?

Reply:
Sure. Do we wanna remove `-fsanitize=signed-integer-overflow` from Speller's `Makefile`?